server/controllers/auth/wikidata_oauth.js

```diff
@@ -3,8 +3,26 @@ const __ = CONFIG.universalPath
 const requests_ = __.require('lib', 'requests')
 const error_ = __.require('lib', 'error/error')
 const root = CONFIG.fullPublicHost()
+const OAuth = require('oauth-1.0a')
+const crypto = require('crypto')
+const createHmacSha1Hash = (baseString, key) => {
+  return crypto.createHmac('sha1', key)
```

Comment:
consider `sha256` => https://github.com/ddo/oauth-1.0a#nodejs

Reply:
unfortunately, when trying with `sha256`, Wikidata answers with `Error: An error occurred in the OAuth protocol: Signature method 'HMAC-SHA256' not supported try one of the following: HMAC-SHA1, RSA-SHA1`